src/views/Space/Treasury.vue

```diff
@@ -31,6 +37,24 @@ const sortedAssets = computed(() =>
   })
 );
 
+const modalState: Ref<{
+  sendToken?: any;
+}> = ref({});
```

Comment:
We don't need that initialState is just for initial state, and here we have none, we can just omit `initialState` prop and get rid of this ref.